patches/node/fix_comment_out_incompatible_crypto_modules.patch

```diff
@@ -9,42 +9,46 @@ with what's exposed through BoringSSL. I plan to upstream parts of this or
 otherwise introduce shims to reduce friction.
 
 diff --git a/src/node_crypto.cc b/src/node_crypto.cc
-index 91cb94d8dbe9db0adbee5e005649188e1ccbcbf9..2429ebf6035de392c6bac18c0b924b995fb3daeb 100644
+index 91cb94d8dbe9db0adbee5e005649188e1ccbcbf9..3eaabf634ab1eb84662243561fa7758601563ef5 100644
 --- a/src/node_crypto.cc
 +++ b/src/node_crypto.cc
-@@ -5191,6 +5191,7 @@ bool DiffieHellman::Init(int primeLength, int g) {
- 
+@@ -5192,11 +5192,11 @@ bool DiffieHellman::Init(int primeLength, int g) {
  bool DiffieHellman::Init(const char* p, int p_len, int g) {
    dh_.reset(DH_new());
-+#if 0
    if (p_len <= 0) {
-     BNerr(BN_F_BN_GENERATE_PRIME_EX, BN_R_BITS_TOO_SMALL);
+-    BNerr(BN_F_BN_GENERATE_PRIME_EX, BN_R_BITS_TOO_SMALL);
++    // BNerr(BN_F_BN_GENERATE_PRIME_EX, BN_R_BITS_TOO_SMALL);
```

Comment:
Commenting out these errors seems bad! Won't that mean we don't get proper error messages in these error cases?

Reply:
no - it means that we get slightly less clean fallback messages, which differ slightly.

if i for example run `crypto.createDiffieHellman(-1)` i see:

<img width="577" alt="Screen Shot 2021-02-09 at 9 53 16 AM" src="https://user-images.githubusercontent.com/2036040/107405898-a9cb4900-6abc-11eb-8e67-e9bd947c1e13.png">

Reply:
I think you've hit the wrong constructor in your test. There are several for DiffieHellman::Init, and this patch does not affect the one-argument version.

Here's an example of how this patch fails to produce an appropriate error:

```
> require('crypto').createDiffieHellman('', 'utf8')
Uncaught Error: Initialization failed
    at new DiffieHellman (internal/crypto/diffiehellman.js:85:19)
    at Object.createDiffieHellman (crypto.js:135:10)
```

Reply:
I've updated the PR to use error routines that work with boringssl.